app/services/CapiService.scala

```diff
@@ -7,13 +7,15 @@ import scala.concurrent.{ExecutionContext, Future}
 class CapiService(apiKey: String, wsClient: WSClient)(implicit val ec: ExecutionContext) {
   private val url = "https://content.guardianapis.com"
 
-  private def request(path: String, querystring: String): Future[String] =
+  private def request(path: String, querystring: Option[String]): Future[String] =
     wsClient
-      .url(s"$url/$path?api-key=$apiKey&$querystring")
+      .url(s"$url/$path?api-key=$apiKey${querystring.map(qs => s"&$qs").getOrElse("")}")
       .get()
       .map(response => response.body)
 
-  def getTags(querystring: String): Future[String] = request("tags", querystring)
+  def getTags(querystring: String): Future[String] = request("tags", Some(querystring))
 
-  def getSections(querystring: String): Future[String] = request("sections", querystring)
+  def getSections(querystring: String): Future[String] = request("sections", Some(querystring))
+
+  def getContent(path: String): Future[String] = request(path, None)
 }
```

Comment:
Can I assume that the changes to `getTags` and `getSections` don't impact on the performance of the existing functionality to fetch and display the tag/section selectors in the Epic forms?

Reply:
just checked in CODE - it's still working!